Src/java/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+plugins {
+    kotlin("jvm") version "2.0.20"
+    id("org.sonarqube") version "4.4.1.3373"
+}
+
+sonar {
+  properties {
+    property("sonar.projectKey", "cqframework_clinical_quality_language")
+    property("sonar.organization", "cqframework")
+    property("sonar.host.url", "https://sonarcloud.io")
+  }
+}
+
```

Comment:
You've removed the `idea` task. If you do not plan to support this anymore, you should also remove it from the [Java README](https://github.com/cqframework/clinical_quality_language/tree/feature-kotlin-build2/Src/java) and [Java Quick Start documentation](https://github.com/cqframework/clinical_quality_language/blob/feature-kotlin-build2/Src/java-quickstart/README.md).

Reply:
IntelliJ supports this natively now, so it's unneeded. I will update the docs.

Reply:
Modulo the grammar, actually. I'll look at that...